docs/HACKING.md

```diff
@@ -238,15 +238,11 @@ You'll need `podman` in addition to `docker`.
 If you used the new `openshift-installer`, it creates a user named `kubeadmin`
 with the password in the file `installer/auth/kubeadmin_password`.
 
-To push the image to the remote registry, set `REMOTE_REGISTRY=true`, `PUSH_USER=kubeadmin`,
-`KUBECONFIG=/path/to/installer/auth/kubeconfig`, and
-`PUSH_PASSWORD=$( cat /path/to/installer/auth/kubeadmin_password )`.  You do not have to
-`oc login` to the cluster.  If you already built the image, use `SKIP_BUILD=true` to do
-a push only.
+To push the image to the remote registry, set `REMOTE_REGISTRY=true`. If you already built
```

Comment:
i dont believe REMOTE_REGISTRY is needed anylonger

Reply:
Hi, @jcantrill you are right. The variable `REMOTE_REGISTRY` appears in `./hack/common`, which is not sourced by `make deploy-image`.